Userland/Libraries/LibDesktop/AppFile.cpp

```diff
@@ -150,12 +150,12 @@ Vector<DeprecatedString> AppFile::launcher_protocols() const
     return protocols;
 }
 
-bool AppFile::spawn() const
+bool AppFile::spawn(Span<DeprecatedString const> arguments) const
```

Comment:
Two things to note from the merge conflict:

- We have `ReadonlySpan<Foo>` now, which is clearer than `Span<Foo const>`, but means the same thing.

- `Core::Process::spawn()` takes StringViews, so `AppFile::spawn()` should too. See next comment...